app/js/show.js

```diff
@@ -37,3 +42,45 @@ for (let i = 0; i < playlistElements.length; i += 1) {
 
     table.appendChild(tableItem);
 }
+
+playButton.onclick = async function () {
+    // value property contains the boolean isPlaying
+    if (playButton.value === 'notPlaying') {
+        playButton.innerText = 'Stop';
+        playButton.value = 'playing';
+
+        playButton.style.backgroundColor = 'red';
+        playButton.style.borderColor = 'red';
+
+        // create show elements with sequence json path
+        showElements = []
+        for (let k = 0; k < table.rows.length; k += 1) {
+            showElements.push(new ShowElementConstructor());
+            showElements[k].setSequenceJson(table.value[k]);
+            var duration = await showElements[k].setUpSequence();
+            showElements[k].setElementLength(duration* 1000)
+        }
+
+        startCanPlay(); // lock to determine ability to play
+        playShow(showElements);
+    } else {
+        playButton.innerText = 'Play';
+        playButton.value = 'notPlaying';
+
+        playButton.style.backgroundColor = 'green';
+        playButton.style.borderColor = 'green';
+
+        stopCanPlay(); // lock to stop play of show
+
+        for (let j = 0; j < showElements.length; j += 1) {
```

Comment:
This does the same thing as lines 83-35. Make a separate function, maybe stopAll()?